### PR TITLE
Create remainder.md, closes #3324 

### PR DESCRIPTION
**Pull request by reganmarie**
### Description

- Created number-format.md on filepath docs/content/go/concepts/math-functions/terms/remainder/remainder.md
- Added Title, Description, Subjects, Tags, and Catalog Content
- Added description, syntax, example, and codebyte example
- Ran through 3 online plagiarism checkers to ensure 100% unique wording and description, passed with 100%
- Followed content and style guidelines -Checked for spelling and grammar errors

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/remainder/remainder.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+Title: 'Remainder()'
+Description: 'Returns the remainder of two given values.'
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/remainder/remainder.md:3]:
```suggestion
Description: 'Returns the remainder of the division of two given values.'
```

File: content/go/concepts/math-functions/terms/remainder/remainder.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Numbers'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Remainder()`** function returns the remainder of two given values. The remainder may be a whole number or the IEEE 754 floating-point.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/remainder/remainder.md:15]:
```suggestion
The **`Remainder()`** function returns the remainder of the division of two given values. The remainder may be a whole number, or a floating-point.
```

File: content/go/concepts/math-functions/terms/remainder/remainder.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Syntax
+
+```pseudo
+result := math.Remainder(num1, num2)
+```
+
+`math.Remainder()` returns the whole or IEEE 754 floating-point remainder of num1 divided by num2, except in five special cases:
+
+- math.Remainder(±Inf, num2) returns NaN
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/remainder/remainder.md:25]:
```suggestion
- If `num1` equals `+-Inf` the function will return `NaN`, if num2 equals `+-Inf` the function returns `num1`
```

File: content/go/concepts/math-functions/terms/remainder/remainder.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+- math.Remainder(NaN, num2) returns NaN
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/remainder/remainder.md:26]:
```suggestion
- If either argument is `NaN` the function will return `NaN`
```

File: content/go/concepts/math-functions/terms/remainder/remainder.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+- math.Remainder(num1, 0) returns NaN
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/remainder/remainder.md:27]:
```suggestion
- If `num2` equals `0` the function will return `NaN`
```

File: content/go/concepts/math-functions/terms/remainder/remainder.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+- math.Remainder(num1, ±Inf) returns num1
+- math.Remainder(num1, NaN) returns NaN
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/remainder/remainder.md:29]:
```suggestion
```

File: content/go/concepts/math-functions/terms/remainder/remainder.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+import (
+  "fmt"
+  "math"
+)
+
+func main() {
+
+  result1 := math.Remainder(4, 102)
+  result2 := math.Remainder(±Inf, 2)
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/remainder/remainder.md:70]:
```suggestion
  result2 := math.Remainder(math.Inf(1), 2)
```

File: content/go/concepts/math-functions/terms/remainder/remainder.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+  fmt.Printf("Result 1: %.1f\n", result_1)
+  fmt.Printf("Result 2: %.1f\n", result_2)
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/remainder/remainder.md:73]:
```suggestion
  fmt.Printf("Result 1: %.1f\n", result1)
  fmt.Printf("Result 2: %.1f\n", result2)
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@reganmarie I've completed my initial review. Please update the entry, and tag me when you've finished making the changes. Thanks

**Comment by reganmarie** [comment]
@caupolicandiaz Updated and committed every revision, listed here! 
- Line 3, change Description to 'Returns the remainder of the division of two given values.'
- Line 15 change to The **`Remainder()`** function returns the remainder of the division of two given values. The remainder may be a whole number, or a floating-point.
- Take out lines 25-29 and replace with combined rules for special cases
- Change Codebyte example in line 70 to result2 := math.Remainder(math.Inf(1), 2)
- Change Codebyte print statements to result1, result2

**Comment by reganmarie** [comment]
Thank you @caupolicandiaz! 😊

**Review by SSwiniarski — APPROVED** [review]
@reganmarie I did a small edit and it looks good to merge.

**Pull request merged**